tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/FolderPickerProducer.java

```diff
@@ -102,7 +102,8 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams, Compon
                     //getting the path of selected folder
                     String[] stringArray = html.split(contentHostingService.getSiteCollection(simplePageBean.getCurrentSiteId()));
                     String[] folderPath = stringArray[1].split("data-files");
-                    UIParameter parameter = new UIParameter("edit-folder-path", folderPath[0].trim());
+                    String path = folderPath[0].trim().replaceAll("'","");
```

Comment:
Why are you doing this replace? If there's a quote in the URL won't it just break?